docs/serving/knative-kubernetes-services.md

```diff
@@ -5,25 +5,36 @@ weight: 9
 type: "docs"
 ---
 
-This document describes what is running when running knative serving.
+This guide describes the components that are active when running Knative Serving.
 
-After applying the serving yaml, this will install a few knative services and
-deployments on your kubernetes cluster. This document provides an overview of
-the deployments and the motivations for each one.
+## Before You Begin
 
-```sh
-$ kubectl get services -n knative-serving
+1. Install Knative Serving. Instructions on how to do this are located 
+   [here](https://knative.dev/docs/install/knative-custom-install/).
+2. Verify that you have the proper components in your cluster. To view the services installed in your cluster, use the commmand:
```

Comment:
```suggestion
2. To view the services installed in your cluster, use the commmand:
```

Reply:
The second part of the sentence is a great intro to the next command. Keep that part! :)